source/2019-11-08-the-ember-times-issue-123.md

```diff
@@ -0,0 +1,149 @@
+Join the EmberConf CFP Brainstorm 🧠⛈️,
+Watch EmberFest from the comfort of your home cinema 🎬,
+
+READMORE
+
+---
+
+## [Section Title in Title Case 🐹](#section-url)
+
+<change section title emoji>
+<consider adding some bold to your paragraph>
+
+<add your name to author list, top and bottom>
+<add blurb and emoji to "SOME-INTRO-HERE">
+
+---
+
+## [Section Title in Title Case 🐹](#section-url)
+
+<change section title emoji>
+<consider adding some bold to your paragraph>
+
+<add your name to author list, top and bottom>
+<add blurb and emoji to "SOME-INTRO-HERE">
+
+---
+
+## [Section Title in Title Case 🐹](#section-url)
+
+<change section title emoji>
+<consider adding some bold to your paragraph>
+
+<add your name to author list, top and bottom>
+<add blurb and emoji to "SOME-INTRO-HERE">
+
+---
+
+## [Section Title in Title Case 🐹](#section-url)
+
+<change section title emoji>
+<consider adding some bold to your paragraph>
+
+<add your name to author list, top and bottom>
+<add blurb and emoji to "SOME-INTRO-HERE">
+
+---
+
+## [Section Title in Title Case 🐹](#section-url)
+
+<change section title emoji>
+<consider adding some bold to your paragraph>
+
+<add your name to author list, top and bottom>
+<add blurb and emoji to "SOME-INTRO-HERE">
+
+---
+
+[## EmberConf CFP Brainstorm 🧠⛈️](https://emberconf.com/#/cfp-brainstorm)
+
+This coming [November 14, 2019 at 10:00am PST](https://everytimezone.com/s/970f2362) the annual Ember CFP Brainstorm is happening over [BlueJeans](https://bluejeans.com/241628685/webrtc)!
+
+The session will go through the content of next year’s [EmberConf](http://emberconf.com/) which is happening between March 16-18, 2020. We'll chat about the CFP, the topics we hope to see, and answer community questions about ideas and proposals. The EmberConf 2020 CFP closes on Dec 1, 2019.
+
+Sign up for a reminder on the [EmberConf CFP Brainstorm website](https://emberconf.com/#/cfp-brainstorm)!
+
+---
+
+## [Section Title in Title Case 🐹](#section-url)
+
+<change section title emoji>
+<consider adding some bold to your paragraph>
+
+<add your name to author list, top and bottom>
+<add blurb and emoji to "SOME-INTRO-HERE">
+
+---
+
+## [Section Title in Title Case 🐹](#section-url)
+
+<change section title emoji>
+<consider adding some bold to your paragraph>
+
+<add your name to author list, top and bottom>
+<add blurb and emoji to "SOME-INTRO-HERE">
+
+---
+
+## [Section Title in Title Case 🐹](#section-url)
+
+<change section title emoji>
+<consider adding some bold to your paragraph>
+
+<add your name to author list, top and bottom>
+<add blurb and emoji to "SOME-INTRO-HERE">
+
+---
+
+## [This Week's Home Cinema Program: EmberFest 2019 🎬🍿](https://www.youtube.com/watch?v=zXbqv2PeYCM&list=PLN4SpDLOSVkT0e094BZhGkUnf2WBF09xx)
+
+This week you can finally start **(re)watching** all of the amazing **21 presentations** about modern Ember from **EmberFest 2019**, which took place on Oct 17 + 18 in Copenhagen, Denmark.
```

Comment:
Anne's lightning talk is currently missing from the playlist. Before we merge this PR, let's check the playlist again to see if we should change this number to 22.